scripts/release_info.py

```diff
@@ -217,8 +218,9 @@ def main():
     with ChangedBranch('develop'):
```

Comment:
Should we be switching to the release branch (not develop) in order to enumerate the logs? Maybe the release branch name could be passed as an argument to the script.

Reply:
Thanks! You're right, we shouldn't poll from develop at all anymore now that we use a different type of release branch. The release documentation mentions this script should only be run from the release branch, so this switching can be removed entirely.

Reply:
Can we do a sanity check that the current branch name is correct (or at least is of the form release-x.x.x) and exit + error if that's the case? In order to prevent errors?

(Just like we check, in setup.sh, that we're in the oppia/ directory before continuing.)

Reply:
Sounds reasonable. Done.